newlib/libc/include/sys/meson.build

```diff
@@ -85,8 +85,10 @@ foreach file : inc_sys_headers_all
   endif
 endforeach
 
-install_headers(inc_sys_headers,
-		install_dir: include_dir / 'sys')
+if really_install
+	install_headers(inc_sys_headers,
```

Comment:
more whitespace; I'll let you find the rest though. Perhaps you can find an editor setting which matches the existing meson indentation?

Reply:
I'm not sure if this is my editor or not (I'm using nano..) but some of the files seem to have tabs, some spaces and I think in some places I saw tabs and spaces and just decided to get it working and fix the whitespace later.

The meson site says: spaces, 2 spaces per tab (https://mesonbuild.com/Style-guide.html#tabs-or-spaces). If I make all the changes follow that rule would it be ok?

Reply:
just use spaces, tabs are a pain